Libraries/MiscDrivers/SRAM/fastspi.c

```diff
@@ -0,0 +1,333 @@
+/******************************************************************************
```

Comment:
Was fastspi used in another example? If so, I think we should give it it's own folder in MiscDrivers

Reply:
I don't even like that it has to exist at all - Ideally I can eventually purge it and use the normal API functions.  Otherwise I agree, it can get its own folder.  For now, though, I want to keep it relatively isolated to the SRAM drivers